docs/src/guides/sat-hunting.md

```diff
@@ -243,7 +243,8 @@ button to display the descriptor.
 
 ### Transferring Ordinals
 
-The `ord` wallet supports transferring specific satoshis. You can also use
-`bitcoin-cli` commands `createrawtransaction`, `signrawtransactionwithwallet`,
-and `sendrawtransaction`, how to do so is complex and outside the scope of this
-guide.
+The `ord` wallet supports transferring specific satoshis by using the 
+name of the satoshi with the `ord wallet send` command. You can also use the
```

Comment:
maybe add an example of this command sending a sat

Reply:
I've updated the doc with an example and more explanation.